docs/generated/packages/jest/executors/jest.json

```diff
@@ -110,7 +110,8 @@
       "runInBand": {
         "alias": "i",
         "description": "Run all tests serially in the current process (rather than creating a worker pool of child processes that run tests). This is sometimes useful for debugging, but such use cases are pretty rare. Useful for CI. (https://jestjs.io/docs/cli#--runinband)",
-        "type": "boolean"
+        "type": "boolean",
+        "default": true
```

Comment:
This is the wrong place, we want to change `@nx/nx-plugin:e2e`

Reply:
thinking on this after a couple minutes. 
if the @nx/nx-plugin:e2e executor doesn't really work without jest set to runInBand. then it would make sense to just leave it hard coded and not let people opt out of it. 
I thought it might have just been a perf thing or whatever. 

If we do add it as an executor option, then probably explain that the tests will probably start failing if you turn this off.

Reply:
Theres ways for it to still work, we need to take another look at our e2e utils